audio-worklet/design-pattern/awn-shared-buffer/shared-buffer-worker.js

```diff
@@ -153,5 +153,8 @@ function initialize(options) {
 onmessage = (eventFromMain) => {
   if (eventFromMain.data.message === 'INITIALIZE_WORKER') {
     initialize(eventFromMain.data.options);
+    return;
```

Comment:
Is this really necessary? No harm in returning early here, not really needed, right?

Reply:
At l.159, the code prints a warning that the "unknown" message is received. So I think the logic is correct.